stories/example-screen-staff-list.stories.js

```diff
@@ -936,19 +719,28 @@ const staff_list_html = `
       <div class="d-flex flex-column justify-content-start flex-wrap w-100 px-2">
         <div class="d-flex justify-content-between align-items-center flex-wrap px-2 w-100">
           <h2>Incident Reports</h2> 
-          <button type="button" class="btn btn-sm btn-danger">
+          <button type="button" class="btn btn-danger fs-6">
             <svg width="1.125rem" height="1.125rem" class="mt-n1 me-1">
               <use xlink:href="static/media/node_modules/@symplicity/tabler-icons/symplicity/tabler-sprite-nostroke.svg#tabler-plus" />
             </svg>Add New
           </button>
         </div>
         <div class="bg-white my-3 w-100">
           <div class="text-center">${tabsDoubleRowHTML}</div>          
-          <div class="p-4">
-            <div style="background-color: #F6F6F6;">
-              <form class="row p-3 m-0 align-items-center">
+            <div style="background-color: #FFF;">
+              <form class="row px-2 py-3 mx-3 mt-2 align-items-center" style="background-color: #F6F6F6;">
                 <div class="col-md-6 col-lg-3 my-1">
-                  <input type="text" class="form-control fs-6" id="text" placeholder="Search cases by student, email, IR#" aria-label="Search">
+                  <div class="input-group">
+                    <div class="input-group-prepend" style="z-index: 5;">
```

Comment:
We should remove this inline style, but I don't know why this would be necessary, can you explain please?